go.mod

```diff
@@ -1,166 +1,161 @@
 module github.com/spiffe/spire
 
-go 1.23.4
+go 1.24.0
```

Comment:
This might be some leftover from rebasing, could you clean it up?

Reply:
@sorindumitru 
this looks up to date.. not sure why github show this diff.
in PR, the go.mod have valid go version